cmd/kubevirt-csi-driver/kubevirt-csi-driver.go

```diff
@@ -23,6 +23,8 @@ var (
 	nodeName               = flag.String("node-name", "", "The node name - the node this pods runs on")
 	infraClusterNamespace  = flag.String("infra-cluster-namespace", "", "The infra-cluster namespace")
 	infraClusterKubeconfig = flag.String("infra-cluster-kubeconfig", "", "the infra-cluster kubeconfig file")
+	infraClusterLabelName  = flag.String("infra-cluster-labelname", "", "The infra-cluster label name to use when creating resources in infra cluster")
```

Comment:
Two separate flags instead of one leaves room for error (set the key and not the value).
I think just a flag for a label is enough, even multi labels, just need validation 
For example `oc label` or `kubectl label` takes `KAY=VAL ... N_KEY=N_VAL` format

Reply:
Currently we have just one label.
These are internal flags for the developer. Developer will test its usage and will not be able to miss the value.
However, if we do change it... do something like:
-label='labelA=valueA,labelB=valueB'
?

Reply:
Done